Builds/containers/packaging/rpm/build_rpm.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -ex
+
+source update_sources.sh
+
+# Build the rpm
+
+IFS='-' read -r RIPPLED_RPM_VERSION RELEASE <<< "$RIPPLED_VERSION"
+export RIPPLED_RPM_VERSION
+
+RPM_RELEASE=${RPM_RELEASE-1}
+
+# post-release version
+if [ "hf" = "$(echo "$RELEASE" | cut -c -2)" ]; then
+    RPM_RELEASE="${RPM_RELEASE}.${RELEASE}"
+# pre-release version (-b or -rc)
+elif [[ $RELEASE ]]; then
+    RPM_RELEASE="0.${RPM_RELEASE}.${RELEASE}"
+fi
+
+export RPM_RELEASE
+
+if [[ $RPM_PATCH ]]; then
+    RPM_PATCH=".${RPM_PATCH}"
+    export RPM_PATCH
+fi
+
+cd rippled
+if [[ -n $(git status --porcelain) ]]; then
+    git status
+    error "Unstaged changes in this repo - please commit first"
+fi
+git archive --format tar.gz --prefix rippled/ -o ../rpmbuild/SOURCES/rippled.tar.gz HEAD
+cd ..
+tar -zc --exclude-vcs -f ./rpmbuild/SOURCES/validator-keys.tar.gz validator-keys-tool/
+
+source /opt/rh/devtoolset-7/enable
+
+rpmbuild --define "_topdir $PWD/rpmbuild" -ba rippled.spec
+rc=$?; if [[ $rc != 0 ]]; then
+    error "error building rpm"
+fi
+
+# Make a tar of the rpm and source rpm
+RPM_VERSION_RELEASE=`rpm -qp --qf='%{NAME}-%{VERSION}-%{RELEASE}' ./rpmbuild/RPMS/x86_64/rippled-[0-9]*.rpm`
+tar_file=$RPM_VERSION_RELEASE.tar.gz
+
+cp ${PWD}/rpmbuild/RPMS/x86_64/* ${PKG_OUTDIR}
+cp ${PWD}/rpmbuild/SRPMS/* ${PKG_OUTDIR}
```

Comment:
Is there a difference between `${PWD}/xxx` and `./xxx`? If not I'd prefer the simplier `.`

Reply:
fixed